docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -0,0 +1,1134 @@
+
+   If this system will use GNOME::
+
+     zfs create                                 rpool/var/lib/AccountsService
+
+   If this system will use Docker (which manages its own datasets &
+   snapshots)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/docker
+
+   If this system will use NFS (locking)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/nfs
+
+   Mount a tmpfs at /run::
+
+     mkdir /mnt/run
+     mount -t tmpfs tmpfs /mnt/run
+     mkdir /mnt/run/lock
+
+   A tmpfs is recommended later, but if you want a separate dataset for
+   ``/tmp``::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/tmp
+     chmod 1777 /mnt/tmp
+
+   The primary goal of this dataset layout is to separate the OS from user
+   data. This allows the root filesystem to be rolled back without rolling
+   back user data.
+
+   If you do nothing extra, ``/tmp`` will be stored as part of the root
+   filesystem. Alternatively, you can create a separate dataset for ``/tmp``,
+   as shown above. This keeps the ``/tmp`` data out of snapshots of your root
+   filesystem. It also allows you to set a quota on ``rpool/tmp``, if you want
+   to limit the maximum space used. Otherwise, you can use a tmpfs (RAM
+   filesystem) later.
+
+#. Install the minimal system::
+
+     debootstrap --components=main,contrib bullseye /mnt 
+
+   The ``debootstrap`` command leaves the new system in an unconfigured state.
+   An alternative to using ``debootstrap`` is to copy the entirety of a
+   working system into the new ZFS root.
+
+#. Copy in zpool.cache::
+
+     mkdir /mnt/etc/zfs
+     cp /etc/zfs/zpool.cache /mnt/etc/zfs/
+
+Step 4: System Configuration
+----------------------------
+
+#. Configure the hostname:
+
+   Replace ``HOSTNAME`` with the desired hostname::
+
+     echo HOSTNAME > /mnt/etc/hostname
+     vi /mnt/etc/hosts
+
+   .. code-block:: text
+
+     Add a line:
+     127.0.1.1       HOSTNAME
+     or if the system has a real name in DNS:
+     127.0.1.1       FQDN HOSTNAME
+
+   **Hint:** Use ``nano`` if you find ``vi`` confusing.
+
+#. Configure the network interface:
+
+   Find the interface name::
+
+     ip addr show
+
+   Adjust ``NAME`` below to match your interface name::
+
+     vi /mnt/etc/network/interfaces.d/NAME
+
+   .. code-block:: text
+
+     auto NAME
+     iface NAME inet dhcp
+
+   Customize this file if the system is not a DHCP client.
+
+#. Configure the package sources::
+
+     vi /mnt/etc/apt/sources.list
+
+   .. code-block:: sourceslist
+
+     deb http://deb.debian.org/debian bullseye main contrib
+     #deb-src http://deb.debian.org/debian bullseye main contrib
+
+     #deb http://security.debian.org/debian-security bullseye/updates main contrib
+     #deb-src http://security.debian.org/debian-security bullseye/updates main contrib
+
+     #deb http://deb.debian.org/debian bullseye-updates main contrib
+     #deb-src http://deb.debian.org/debian bullseye-updates main contrib
```

Comment:
those have changed in bullseye, the security sources will be `deb https://deb.debian.org/debian-security bullseye-security main contrib`